### ISSUE TITLE
Sipi Configuration file not found

### ISSUE THREAD

**Issue by janCstoffregen** [nie-ine/Quick-Installation-Guide---Knora-Salsah-Sipi-GraphDB#5]
sipi_1     | Configuration file not found: /sipi/config/sipi.knora-docker-config.lua
sipi_1     |   --config filename, -c filename      Configuration file for web server.
sipi_1     | 
quickinstallationguideknorasalsahsipigraphdb_sipi_1 exited with code 1

**Comment by janCstoffregen** [comment]
Regarding the problem with Sipi:

I found that in docker-composer.yml you used "Prebuiltimages" (with lower "i" for images) instead of "PrebuiltImages", so I changed it:
- ./PrebuiltImages/sipiConfig:/sipi/config
but now I receive another error for Sipi:
sipi_1     | Error at [/sipi/src/SipiFilenameHash.cpp: 104]: Inconsistent directory tree!

Where else should I change it?

roberta

**Comment by janCstoffregen** [comment]
@Padlina zur Info

**Comment by janCstoffregen** [comment]
Hi Roberta,
you have to run docker-compose up in the Prebuiltimages - Folder, then it should work… Otherwise you built everything from scratch again which takes quite a while and is error intensive..

But then you should not have the mapping problem anymore either.. 

Does it work?

Thanks and best regards,
Jan

**Comment by Padlina** [comment]
yes, it works from the PrebuilImages folder. and the import.py script works as well.
But I thought that to have the last version of Knora we should run docker-compose up from the root folder. Is this right?

**Comment by janCstoffregen** [comment]
Not anymore because I updated the docker images on our docker hub, so if you run the docker-compose up command in the prebuilt folder now it pulls the latest version of Knora.

**Comment by Padlina** [comment]
Ok, perfect then. I'll try the bulk import and let you know.

**Comment by janCstoffregen** [comment]
ok, thanks, I'll leave this issue open until I fix the volume mapping though.
